src/TodoListItem.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 
-function TodoListItem({key, todo}){
+function TodoListItem({key, todo, onRemoveTodo}){
    return(
     <li key={key}>
         {todo} 
```

Comment:
Hmm you're trying to display the object here.
You want title? Try and figure out how to display the title here